### ISSUE TITLE
Rate Limits

### ISSUE THREAD

**Issue by Fmacmak** [SV3A/bBucket2gHub#10]
When syncing from a workspace with lots of repositories and lots of commits, there is a high possibility of being rate limited by Bitbucket and/or Github.

[Github limits
](https://docs.github.com/rest/overview/rate-limits-for-the-rest-api)

[Bitbucket limits](https://support.atlassian.com/bitbucket-cloud/docs/api-request-limits/)

An option/flag to manually list the repos to sync would help. Alongside a timeout to prevent being rate limited.

**Comment by RhymezxCode** [comment]
Yeah, this is true I was only able to push 270 commits before I was hit with a socket timeout, and I cannot continue except I delete the repo and start again.
